src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -117,13 +122,49 @@ namespace RSS
                     && (lastMatch == other.lastMatch)
                     && (savePath == other.savePath)
                     && (category == other.category)
-                    && (addPaused == other.addPaused);
+                    && (addPaused == other.addPaused)
+                    && (smartFilter == other.smartFilter);
         }
     };
 }
 
 using namespace RSS;
 
+QString computeEpisodeName(const QString &article)
+{
+    static const QRegularExpression episodeRegex(
```

Comment:
actually no, but if there is no performance bottleneck, I would like to keep the code simple (and avoid premature optimization), I'm talking about person preference here.